### PR TITLE
Add to user.md

### PR DESCRIPTION
**Pull request by OrionStar25**
This pull request is for the issue no. #3 

I make sure that before opening this pull request I have starred this repo . 

Live link of gh pages for my fork is:  
[Live link](orionstar25.github.io/awesome-first-timers)

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-This pull request is for the issue no. #issue_no  
+This pull request is for the issue no. #4  
 
-I make sure that before opening this pull request I have starred this repo  
+I make sure that before opening this pull request I have starred this repo.  
 
 Live link of gh pages for my fork is:  
-[Live link](provide live link here)  
+[Live link](https://github.com/OrionStar25/awesome-first-timers)
```
Comment (Anupam-dagar) [inline PULL_REQUEST_TEMPLATE.md]: This file doesn't need to be changed.Please undo the changes.

File: users.md [hunk 2]
```diff
@@ -267,3 +267,8 @@
   Place: Japan  
   Bio: PhD student in Software Engineering  
   Github: [sruangwan](https://github.com/sruangwan)  
+
+- Name: Niharika Shrivastava  
+  Place: India  
+  Bio: IT Student  
+  Github: [OrionStar25](https://github.com/OrionStar25)
```
Comment (Anupam-dagar) [inline users.md]: LGTM :+1:

**Comment by Anupam-dagar** [comment]
Undo the changes in PULL_REQUEST_TEMPLATE.md and I'll merge.

**Comment by OrionStar25** [comment]
Check now (?)
